File: app/src/main/java/ai/chench/assidua/Expenditure.java
package ai.chench.assidua;

import java.math.BigDecimal;
import java.util.Date;

public class Expenditure {
    private BigDecimal value;
    private String name;
    private Date date;

    public Expenditure(String name, BigDecimal value, Date date) {
        this.name = name;
        this.value = value;
        this.date = date;
    }

    public BigDecimal getValue() {
        return value;
    }
    public void setValue(BigDecimal value) {
        this.value = value;
    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public Date getDate() {
        return date;
    }
    public void setDate(Date date) {
        this.date = date;
    }
}

File: app/src/main/java/ai/chench/assidua/Expenditure.kt
package ai.chench.assidua

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.math.BigDecimal
import java.util.*

@Entity(tableName = "expenditure_table")
data class Expenditure(
        var name: String,
        var value: BigDecimal,
        var date: Date,
        var budgetId: UUID,
        var id: UUID = UUID.randomUUID())

